### PR TITLE
Feature/merged state mgmt and rtids to picklists  updates

### PR DESCRIPTION
**Pull request by kenlewis-sfdo**
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by kenlewis-sfdo** [comment]
Hey @psadie-sfdo will you take a quick look at these additional changes to our combined pr?  Thanks!

**Review by psadie-sfdo — APPROVED** [review]
Looks good to me!

File: src/lwc/geFormRenderer/geFormRenderer.js [hunk 1]
```diff
@@ -1199,36 +1199,32 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
...
+        if (fieldApiName === DATA_IMPORT_ACCOUNT1_IMPORTED_FIELD.fieldApiName) {
             this.handleDonorAccountChange(recordId);
-        } else if (fieldApiName === contact1Imported) {
+        } else if (fieldApiName === DATA_IMPORT_CONTACT1_IMPORTED_FIELD.fieldApiName) {
             this.handleDonorContactChange(recordId);
         }
     }
 
+    objectMappingDeveloperNameFor(fieldApiName) {
+        return this.getObjectMapping(fieldApiName).DeveloperName;
```
Comment (psadie-sfdo) [inline src/lwc/geFormRenderer/geFormRenderer.js:1225]: Should we add an undefined/null check for this?

Comment (kenlewis-sfdo) [reply src/lwc/geFormRenderer/geFormRenderer.js]: Yep - done!

**Pull request merged**